client/src/components/Notices/NoticeDetail.js

```diff
@@ -39,11 +48,11 @@ const _NoticeDetail = ({notice}: NoticeDetailProps) => {
     </span>
   ) : null
 
-  const estimate = notice.price_est_low ? (
+  const estimate = notice.price_est_high !== null ? (
```

Comment:
I prefer using lodash's `isNil`, because maybe it can be undefined sometimes, or?

Reply:
Oh, and is the change from `low` to `high` intended? I'm not sure based on the PR's description.

Reply:
I forgot `isNil` existed :) change from `low` to `high` was first try at fixing the problem, but i decided checking with `isNill` was more explicit, and `high` stayed.